provider/test/provider.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	"github.com/go-spatial/geom"
-	"github.com/go-spatial/tegola/proj"
+	"github.com/go-spatial/proj"
```

Comment:
I know it's functionally not different, but please revert this import to the `tegola/proj` package so we're consistent.